docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -140,28 +140,28 @@ function setData(
 ) public
 ```
 
-Set data as **bytes** in the account storage for a single key.
+Sets data in the account storage for a particular key as **bytes**.
 
-_Triggers the **[DataChanged](#datachanged)** event when setting data successfully._
+_Triggers the **[DataChanged](#datachanged)** event when successfully setting the data ._
 
 :::note
-**The `setData(...)` function can only be called by the current owner of the contract.**
+The `setData(...)` function can only be called by the current owner of the contract.
 :::
 
 #### Parameters:
 
-| Name    | Type    | Description                    |
-| :------ | :------ | :----------------------------- |
-| `key`   | bytes32 | The key for which to set data. |
-| `value` | bytes   | The data to set as bytes.      |
+| Name    | Type    | Description                             |
+| :------ | :------ | :-------------------------------------- |
+| `key`   | bytes32 | The key for which you want to set data. |
+| `value` | bytes   | The data you want to set as bytes.      |
 
 ### getData
 
 ```solidity
 function getData(bytes32 key) public view returns (bytes memory value)
 ```
 
-Retrieve the data set for the given key.
+Is called to retrieve the data set for the given key.
```

Comment:
Can we please always use present tense / 3rd person singular for the API docs.

Reply:
agree